xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -857,7 +857,7 @@ namespace XBMCAddon
       for (const auto& el : alt.later())
       {
         if (el.which() == second)
-          throw WrongTypeException(StringUtils::Format("When using \"%s\" you need to supply a string or list of strings for the value in the dictionary", tag.c_str()).c_str());
+          throw WrongTypeException("%s", StringUtils::Format("When using \"%s\" you need to supply a string or list of strings for the value in the dictionary", tag.c_str()).c_str());
```

Comment:
I have no idea, but Stringutils has some platform specific handling and was used here before, so I didn't want to unwillingly change behavior.
I can change it to remove its use if preferred.